internalshared/configutil/kms.go

```diff
@@ -99,6 +100,15 @@ func parseKMS(result *[]*KMS, list *ast.ObjectList, blockName string, maxKMS int
 			delete(m, "disabled")
 		}
 
+		var recover bool
+		if v, ok := m["recovery_mode"]; ok {
```

Comment:
I worry a tiny bit about confusion with the existing "recovery mode" in vault: https://developer.hashicorp.com/vault/docs/concepts/recovery-mode.  Suggested alternative: enable_recovery_keys?

Reply:
Yeah, I was struggling with verbiage too.  I was also considering `recovery_keys_fallback`.

Reply:
That seems better, less ambiguous than my suggestion.